do/droplets.go

```diff
@@ -93,6 +95,7 @@ func (i *instances) ExternalID(nodeName types.NodeName) (string, error) {
 func (i *instances) InstanceID(nodeName types.NodeName) (string, error) {
 	droplet, err := i.dropletByName(context.TODO(), nodeName)
 	if err != nil {
+		glog.Infof("InstanceID %v : %v", nodeName, droplet.ID)
```

Comment:
Actually I think all `glog` uses in the PR contain debug info that might have been useful for development, but that might not add valuable info at runtime.
So, same as above, probably a different PR (previous issue) and changing logging level for these comments might be a choice.

Reply:
if we already have an error, i think its better to see in which context it happened, i just might merge it together with the error message, to avoid duplicate logs.